ttl_manager.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2023 The nutsdb Author. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nutsdb
+
+import (
+	"github.com/antlabs/timer"
+	"time"
+)
+
+type ttlManager struct {
+	t          timer.Timer
+	timerNodes map[string]map[string]timer.TimeNoder
```

Comment:
This struct looks too complex. Please mark it as a type and add more methods to it. It will looks more elegant.